nav2_behavior_tree/include/nav2_behavior_tree/bt_service_node.hpp

```diff
@@ -59,10 +60,6 @@ class BtServiceNode : public BT::CoroActionNode
   {
     node_ = blackboard()->template get<rclcpp::Node::SharedPtr>("node");
 
-    // Get the required items from the blackboard
```

Comment:
This is a necessary variable, please readd

Reply:
done